src/bentoml/_internal/utils/circus/__init__.py

```diff
@@ -53,5 +53,9 @@ def create_standalone_arbiter(watchers: list[Watcher], **kwargs: t.Any) -> Arbit
                 watchers,
                 endpoint=f"tcp://127.0.0.1:{endpoint_port}",
                 pubsub_endpoint=f"tcp://127.0.0.1:{pubsub_port}",
+                # NOTE: increase check delay to 5s to avoid
+                # ConfclitError raised when arbiter is running arbiter_stop command
+                # See https://github.com/bentoml/BentoML/pull/3540.
+                check_delay=5,
```

Comment:
I think this could use some more explanation, either here or in the PR description.